contracts/Manager.sol

```diff
@@ -209,6 +219,73 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
      * groups will be the first to have their votes withdrawn.
      */
     function deprecateGroup(address group) external onlyOwner {
+        _deprecateGroup(group);
+    }
+
+    function isGroupMemberElected(address groupMember) private view returns (bool) {
+        IElection election = getElection();
+
+        address[] memory electedValidatorSigners = election.electValidatorSigners();
+
+        for (uint256 i = 0; i < electedValidatorSigners.length; i++) {
+            if (electedValidatorSigners[i] == groupMember) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    function isValidGroup(address group) private view returns (bool) {
+        IValidators validators = getValidators();
+
+        // add check if group is !registered
+        if (!validators.isValidatorGroup(group)) {
+            return false;
+        }
+
+        (address[] memory members, , , , , uint256 slashMultiplier, ) = validators
+            .getValidatorGroup(group);
+
+        // check if group has no members
+        if (members.length == 0) {
+            return false;
+        }
+        // check for recent slash
+        if ((slashMultiplier) < 10**24) {
+            return false;
+        }
+        // check for majority of members are elected.
```

Comment:
Wondering if it is necessary to check if the majority of the members are elected, vs only checking if at least one member is elected.

The original thought was that having more elected validator in a groups brings more reward, but there is ultimately no penalty to only having one member that is elected.

So only checking that at least 1 member is elected might save us some gas.

Reply:
@martinvol thoughts?

Reply:
After chatting with @martinvol we decided to check if at least 1 member is elected. Changes were made in #75